website/docs/maintenance/backup_restore.md

```diff
@@ -49,7 +49,7 @@ CREATE EXTENSION earthdistance WITH SCHEMA public;
 .
 
 # Restore
-docker compose exec -T database psql -U teslamate -d teslamate < teslamate.bck
+docker compose exec -T database psql -U teslamate -d teslamate < teslamate.bck.zstd
```

Comment:
This will result in

```bash
The input is a PostgreSQL custom-format dump.
Use the pg_restore command-line client to restore this dump to a database.
```